From ba3cf9fd1a086ae69f5f5e1c63f093847804dd9d Mon Sep 17 00:00:00 2001
From: "Nguyen Van Linh [FGA.BU13]" <LinhNV25@fsoft.com.vn>
Date: Thu, 29 Mar 2018 11:26:55 +0700
Subject: [PATCH 245/504] vl4: vsp1: Fix mistake of vsp1_pipeline_stopped()
 function

This patch replace comma with a semicolon in line
to fix mistake 196 into vsp1_pipeline_stopped() function
Refer to patch "v4l: vsp1: Split pipeline management code from
vsp1_video.c" for origin source code.

Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNV25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 drivers/media/platform/vsp1/vsp1_pipe.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/vsp1/vsp1_pipe.c b/drivers/media/platform/vsp1/vsp1_pipe.c
index facf68f..a33faa8 100644
--- a/drivers/media/platform/vsp1/vsp1_pipe.c
+++ b/drivers/media/platform/vsp1/vsp1_pipe.c
@@ -193,7 +193,7 @@ bool vsp1_pipeline_stopped(struct vsp1_pipeline *pipe)
 	bool stopped;
 
 	spin_lock_irqsave(&pipe->irqlock, flags);
-	stopped = pipe->state == VSP1_PIPELINE_STOPPED,
+	stopped = pipe->state == VSP1_PIPELINE_STOPPED;
 	spin_unlock_irqrestore(&pipe->irqlock, flags);
 
 	return stopped;
-- 
2.7.4

